lib/schedule/data.ex

```diff
@@ -269,6 +269,7 @@ defmodule Schedule.Data do
     gtfs_trips = Gtfs.Trip.parse(gtfs_files["trips.txt"], bus_route_ids)
     gtfs_trip_ids = MapSet.new(gtfs_trips, & &1.id)
     stop_times_by_id = StopTime.parse(gtfs_files["stop_times.txt"], gtfs_trip_ids)
+
```

Comment:
Diff seems extraneous to the rest of the PR, may have been committed accidentally but I'm not sure how much removing it matters.

Reply:
eh, it takes one extra second to remove it, and it keeps the commit history that much cleaner